software/utils/drc_utils/python/send_rotate.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+import os,sys
+import time
+import datetime
+import subprocess
+
+home_dir =os.getenv("HOME")
+#print home_dir
+sys.path.append(home_dir + "/drc/software/build/lib/python2.7/site-packages")
+sys.path.append(home_dir + "/drc/software/build/lib/python2.7/dist-packages")
```

Comment:
7-10 are redundant in my opinion if drc_environment is sourced as part of the bashrc or manually, especially since it does not live at ~/drc anymore for most people